dev/benchmarks/platform_channels_benchmarks/lib/main.dart

```diff
@@ -48,60 +50,105 @@ List<Object> _makeTestBuffer(int size) {
   return answer;
 }
 
-Future<void> _runTests() async {
-  if (kDebugMode) {
-    throw Exception(
-        "Must be run in profile mode! Use 'flutter run --profile'.");
+Future<double> _runBasicStandardSmall(
+    BasicMessageChannel<Object> basicStandard) async {
+  final Stopwatch watch = Stopwatch();
+  watch.start();
```

Comment:
For this and the one below, we should have a warmup lap. Even though the Dart code is AOT in profile/release modes, on at least the Android side we'll be hitting JVM code.

Reply:
Nice catch before these are turned on, done.

Reply:
Did you add/commit the changes? I'm not seeing it yet.

Reply:
And in case it's not clear, what I mean by warmup here is to actually use the message channel at least once before you start the stop watch.

Reply:
Yep it's there.  Each function is called with the parameter of `count  = 1` before it is actually measured.

Reply:
The stopwatch happens for the warmup, but the result is thrown away

Reply:
Ahhh ok, thanks I missed that.